config/management/genesis/src/command.rs

```diff
@@ -8,61 +8,53 @@ use structopt::StructOpt;
 #[derive(Debug, StructOpt)]
 #[structopt(about = "Tool used for genesis")]
 pub enum Command {
-    #[structopt(about = "Submits an Ed25519PublicKey for the libra root")]
-    LibraRootKey(libra_management::key::LibraRootKey),
-    #[structopt(about = "Create a waypoint and optionally place it in a store")]
-    CreateWaypoint(libra_management::waypoint::CreateWaypoint),
+    #[structopt(about = "Create a waypoint and place it in a store")]
```

Comment:
We use `storage` in operational, and `store` in `genesis`.  We should probably do another pass to keep things similar.

Reply:
yeah, I agree.